Calling/ClientApp/src/Utils/Utils.ts

```diff
@@ -73,3 +73,57 @@ export const utils = {
     return dateTimeStamp;
   }
 };
+
+/* Chat */
+export const compareMessages = (firstMessage: any, secondMessage: any) => {
```

Comment:
nit: Looks like we are not referencing these avatars, will it be good idea to clean up these, if you are not planning to use these in upcoming updates?

Reply:
I left them as placeholders in case they want avatars.